plugins/permission-backend/src/service/router.test.ts

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright 2021 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import express from 'express';
+import request from 'supertest';
+import { getVoidLogger } from '@backstage/backend-common';
+import {
+  AuthorizeResult,
+  Permission,
+} from '@backstage/plugin-permission-common';
+import { PermissionPolicy } from '@backstage/plugin-permission-node';
+
+import { createRouter } from './router';
+
+jest.mock('@backstage/plugin-auth-backend', () => {
```

Comment:
ooh maybe this client should be passed in as a constructor argument instead, so you don't have to mock an entire module like this?

Reply:
Yeah, makes sense! Done in 12b0eebfb5. Note that we still pull in the IdentityClient to use the static `getBearerToken` method, but there's no need to mock that, so we get to mock less stuff :tada:.